src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -287,14 +323,28 @@ void emberAfPumpConfigurationAndControlClusterServerInitCallback(EndpointId endp
     emberAfDebugPrintln(
         "Constant Pressure %s",
         FeatureSupportedDebugString(endpoint, Attributes::MinConstPressure::Get, Attributes::MaxConstPressure::Get));
-    emberAfDebugPrintln("Constant Proportional Pressure %s",
+    emberAfDebugPrintln("PCC Server: Constant Proportional Pressure %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinCompPressure::Get, Attributes::MaxCompPressure::Get));
-    emberAfDebugPrintln("Constant Flow %s",
+    emberAfDebugPrintln("PCC Server: Constant Flow %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinConstFlow::Get, Attributes::MaxConstFlow::Get));
-    emberAfDebugPrintln("Constant Temperature %s",
+    emberAfDebugPrintln("PCC Server: Constant Temperature %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinConstTemp::Get, Attributes::MaxConstTemp::Get));
-    emberAfDebugPrintln("Constant Speed %s",
+    emberAfDebugPrintln("PCC Server: Constant Speed %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinConstSpeed::Get, Attributes::MaxConstSpeed::Get));
+
+    status = Attributes::ControlMode::Get(endpoint, &controlMode);
```

Comment:
Why is this using a get instead of `emberAfContainsAttribute`?

Reply:
I had not spotted that function, but it clearly seems to serve the purpose I'm looking so I will get that updated

Reply:
I updated the implementation to use the emberAfContainsAttribute function to determine if the attribute is available, thanks for pointing me to that function 👍🏻